src/entities/note.cpp

```diff
@@ -2277,17 +2277,36 @@ QString Note::textToMarkdownHtml(QString str, const QString &notesPath,
        src=\")((?!file:\/\/).+)\")"),
                 "\\1file://" + windowsSlash + notePath + "/\\2\"");
     */
-
-    const QString fontString =
-        settings.value(QStringLiteral("MainWindow/noteTextView.code.font"))
+    const QString previewFontString =
+        settings.value(QStringLiteral("MainWindow/noteTextEdit.font"))
             .toString();
+    QString margin_listitem_str = "";
+    QString margin_checkbox_str = "";
+    if (!previewFontString.isEmpty()) {
+        QFont previewFont;
+        previewFont.fromString(previewFontString);
+        const int width_checkbox = QFontMetrics(previewFont).horizontalAdvance("☑"); // &#9744; and &#9745;
```

Comment:
You also need to take the size of the unchecked box, since the width can be different.

Reply:
And can you please use the same variable style as in the rest of the code (camelCase).

Reply:
Are you sure? That would mean checked/unchecked need different style classes.

Reply:
My screenshot says so. They are two different characters. Every character can have different widths.